### ISSUE TITLE
Does not work with latest version of Chrome (version 66) and Safari 

### ISSUE THREAD

**Issue by DenisDyack** [pupunzi/jquery.mb.YTPlayer#357]
Hi There,

This plugin no longer works with the latest version of Chrome and Safari - it was working previously for several months until we updated our browsers.

You can check the site out [here](url = https://apocalypse333.com).

It seems work intermittently and if you get to a 404.html on the page and return - then it will work.  So depending on redirections, it may work.  (We don't do much web work, so we are not sure).

It also does not seem to work with Safari.

It does work with FireFox and Edge.  We tested this from several machines and different locations site over the globe.

I hope this helps!

**Comment by DenisDyack** [comment]
Also - here is an error we have gotten:

www-widgetapi.js:99 Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://www.youtube.com') does not match the recipient window's origin ('https://www.apocalypse333.com').
h.B @ www-widgetapi.js:99

**Comment by Le-future** [comment]
The solution provided by @DenisDyack is fine, except one point, how to play the video with the sound? I tried to fire it up muted, that works, then applied the .YTPUnmute(); in a setTimeout function, but immediately after that, the video stops 😞

**Comment by Le-future** [comment]
What about to dedicate a button in the HTML for mute/unmute and simulate a click on it through JS?

**Comment by pupunzi** [comment]
The simulated event doesn't the trick it needs a physic action

**Comment by Le-future** [comment]
I just added a button in my HTML to unmute the video, but it seems that now, even these functions are no longer working:

jQuery.fn.YTPMute()
jQuery.fn.YTPUnmute()
jQuery.fn.YTPToggleVolume()

**Comment by pupunzi** [comment]
Hi,
Those methods are working fine; check this demo:
https://pupunzi.com/mb.components/mb.YTPlayer/demo/demo.html

**Comment by hypersonic125** [comment]
Hello,
With all respect, even in your demo, let's say at Change Video 2, if you press Pause in the middle of the movie the youtube controls appear, the movie title etc. It is not nice for a background. It is possible to solve this maybe with a fall-back photo when the movie stops?

**Comment by pupunzi** [comment]
@hypersonic125 Are you sure it doesn't depend on your Chrome browser extension?
I just tested it on Chrome latest version (Version 67.0.3396.62) without any problem.
here is a screencast:

https://pupunzi.com/downloads/ytp.mp4

**Comment by hypersonic125** [comment]
Yes, I just updated the Chrome. Please be so kind and check yourself, it is not a secret, it is the home page of www.tornado.ro, The settings for the background video are autoplay, mute:true. It starts but if I press pause the ugly writing appears. I have some other movies in header pages or in smaller frames and it is the same. The player was provided with the Bootstrap, Titan template from Theme wagon. Do I have to replace the player that was provided with the template maybe with a newer version? The ugly part is that I work a lot for the design and everything was just perfect, I was very happy with player. In the last two weeks when I just launched the site presenting it to our resellers with big announcements and a lot of noise it started to show the youtube stuff :-( :-(

**Comment by pupunzi** [comment]
Hi,
I think is some extension you've on your Chrome browser. It doesn't happen to me also on your site:

https://pupunzi.com/downloads/ytp-2.mp4
